sefaz/accesskey/errors.go

```diff
@@ -9,10 +9,26 @@ var (
 	ErrCodeInvalidAccessKey = errors.Code("INVALID_ACCESS_KEY")
```

Comment:
If the error codes became more specific, then the generic "INVALID_ACCESS_KEY" error should be removed, because it could be confusing when each error is raised

Reply:
The point here was to keep backward compatibility with the previous version. If we simply remove this code, that would be a breaking change once this is an Exported symbol.

Another approach would be deprecating this generic error code, what do you think?

Reply:
`ErrCodeInvalidAccessKey` could still be defined and not deprecated as a means of grouping all access keys validation errors under a common code. It's expected that most users will want to return an `INVALID_ACCESS_KEY` code instead of many detailed codes, and if there's not a common defined code each user would create its own `INVALID_ACCESS_KEY`, `BAD_ACCESS_KEY`, `BAD_AK`, `INVALID_AK`, etc

Reply:
It makes sense. @andremissaglia what is your opinion about it?

Reply:
Just some clarification. The idea behind error codes is to make easy for handling errors.  For example: if the error was caused by an invalid month it will be ignored but if the error was caused by invalid cnpj we would abort the operation. So we would just make a switch by error code.
If we have a large number of error codes, an HTTP layer, for example, might have to check for all of then to set an proper response.